src/view/dialog/NyDialog.tsx

```diff
@@ -43,10 +44,21 @@ function NyDialog() {
     const dialoger = useDialogContext();
     const bruker = useUserInfoContext();
     const history = useHistory();
+    const kanSendeMeldign = useKansendeMelding();
 
     const [ferdigBehandlet, setFerdigBehandlet] = useState(true);
     const [venterPaSvar, setVenterPaSvar] = useState(false);
 
+    if (!kanSendeMeldign) {
+        return (
+            <div className="dialog dialog-new">
+                Du kan ikke sende dialog
+                <br />
+                Se feilmelding over
```

Comment:
Syns ikke den her helt tilfører noen verdi. Ville heller bare latt den stå tom eller snakket med Linn for å få noen skisser. Alertstripa står jo øverst og det er litt spesielt å ha navigert rett til /ny